binaries/cuprated/src/rpc/request/address_book.rs

```diff
@@ -0,0 +1,104 @@
+//! Functions for TODO: doc enum message.
+
+use std::convert::Infallible;
+
+use anyhow::Error;
+use tower::ServiceExt;
+
+use cuprate_helper::cast::usize_to_u64;
+use cuprate_p2p_core::{
+    services::{AddressBookRequest, AddressBookResponse},
+    AddressBook, NetworkZone,
+};
+
+/// [`AddressBookRequest::PeerlistSize`]
+pub(super) async fn peerlist_size<Z: NetworkZone>(
+    address_book: &mut impl AddressBook<Z>,
+) -> Result<(u64, u64), Error> {
+    let AddressBookResponse::PeerlistSize { white, grey } = address_book
+        .ready()
+        .await
+        .expect("TODO")
+        .call(AddressBookRequest::PeerlistSize)
+        .await
+        .expect("TODO")
+    else {
+        unreachable!();
+    };
+
+    Ok((usize_to_u64(white), usize_to_u64(grey)))
+}
```

Comment:
### Misc
The services that use `tower::BoxError` don't work with `anyhow::Error` when using `?`, can we swap them to `anyhow`?

The services are:
- address book
- blockchain context
- transaction pool

Reply:
Yeah, there are other as well, probably should be done in a separate PR